internal/testutils/jimmtest/suite.go

```diff
@@ -47,7 +46,11 @@ func (t GocheckTester) Name() string {
 }
 
 func (t GocheckTester) Cleanup(f func()) {
-	t.C.Logf("warning: gocheck does not support Cleanup functions; make sure you're using suite's tear-down method")
+	if t.AddCleanup != nil {
+		t.AddCleanup(f)
```

Comment:
why bother with adding the cleanup? Since we don't have any cleanup with all the `ŧesting.t` tests.

I think it's just enough to increase the conn limit on postgres

Reply:
btw, where did the change go? Or now it's fine because we are not creating that many connections?

Reply:
https://github.com/canonical/jimm/pull/1468/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9R73

Reply:
I think this link stops working when you force push btw.